custom/preact/archive-all-email-recs/maintenance.preact.archive-all-email-recs.yml

```diff
@@ -0,0 +1,7 @@
+version: archive-all-email-recs
+description: 'archives and removes all email recommendations'
+
+steps:
+    - run: |
+        chmod u+x ./patch/archive-all-email-recs.sh
```

Comment:
This isn't hurting anything, but I'm not sure this is necessary - we were able to use a similar script previously without the need:
https://github.com/searchspring/snapfu-patches/blob/main/preact/0.45.1/maintenance.preact.0.45.1.yml